SPECS/python-ptyprocess/python-ptyprocess.spec

```diff
@@ -0,0 +1,141 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%global srcname ptyprocess
+Summary:        Run a subprocess in a pseudo terminal
+Name:           python-%{srcname}
+Version:        0.7.0
+Release:        12%{?dist}
```

Comment:
The `Release` tag doesn't match the changelog.

Reply:
Fixed, thanks!